### PR TITLE
Logo and Metadata Addition

### PR DESCRIPTION
**Pull request by sahsisunny**
## Issue Ticket Number

-   #86

## Description

This pull request introduces enhancements to the branding by adding a logo and metadata. The changes aim to improve the overall look and feel of the application.

**Breaking Changes**

-   [ ] Yes
-   [x] No

**Development Tested?**

-   [x] Yes
-   [ ] No

**Tested in Staging?**

-   [x] Yes
-   [ ] No

**Under Feature Flag**

-   [x] Yes
-   [ ] No

**Database Changes**

-   [x] No
-   [ ] Yes

### Screenshots

image
## Test Coverage

File                          | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
------------------------------|---------|----------|---------|---------|-------------------
All files                     |   97.36 |     92.1 |   93.67 |   98.45 |                   
 fixtures                     |     100 |      100 |     100 |     100 |                   
  users.ts                    |     100 |      100 |     100 |     100 |                   
 src/components/App           |     100 |      100 |     100 |     100 |                   
  InputSection.tsx            |     100 |      100 |     100 |     100 |                   
  OutputSection.tsx           |     100 |      100 |     100 |     100 |                   
 src/components/Button        |     100 |      100 |     100 |     100 |                   
  index.tsx                   |     100 |      100 |     100 |     100 | 
 src/components/Dashboard     |     100 |      100 |     100 |     100 | 
  NoUrlFound.tsx              |     100 |      100 |     100 |     100 | 
  UrlList.tsx                 |     100 |      100 |     100 |     100 | 
  UrlListItem.tsx             |     100 |      100 |     100 |     100 | 
 src/components/Footer        |     100 |      100 |     100 |     100 | 
  index.tsx                   |     100 |      100 |     100 |     100 | 
 src/components/Layout        |     100 |      100 |     100 |     100 | 
  index.tsx                   |     100 |      100 |     100 |     100 | 
 src/components/LoginModal    |     100 |      100 |     100 |     100 | 
  index.tsx                   |     100 |      100 |     100 |     100 | 
 src/components/Navbar        |   94.11 |      100 |   71.42 |   93.54 | 
  NavbarMenuItems.tsx         |     100 |      100 |     100 |     100 | 
  UserProfileButton.tsx       |     100 |      100 |     100 |     100 | 
  index.tsx                   |   90.47 |      100 |      50 |      90 | 19,54
 src/components/ProfileIcon   |     100 |      100 |     100 |     100 | 
  ProfileIcon.tsx             |     100 |      100 |     100 |     100 | 
 src/components/Redirect      |     100 |      100 |     100 |     100 | 
  ErrorPage.tsx               |     100 |      100 |     100 |     100 | 
  LoaderTimer.tsx             |     100 |      100 |     100 |     100 | 
  NotFound.tsx                |     100 |      100 |     100 |     100 | 
  RedirectFooter.tsx          |     100 |      100 |     100 |     100 | 
 src/components/ShimmerEffect |     100 |      100 |     100 |     100 | 
  DashboardShimmer.tsx        |     100 |      100 |     100 |     100 | 
  OutputSectionShimmer.tsx    |     100 |      100 |     100 |     100 | 
  RedirectShimmer.tsx         |     100 |      100 |     100 |     100 | 
  UserLoginShimmer.tsx        |     100 |      100 |     100 |     100 | 
 src/components/Toast         |     100 |      100 |     100 |     100 | 
  index.tsx                   |     100 |      100 |     100 |     100 | 
 src/components/icons         |     100 |      100 |     100 |     100 | 
  redirect.tsx                |     100 |      100 |     100 |     100 | 
  signWithGoogle.tsx          |     100 |      100 |     100 |     100 | 
 src/constants                |   89.47 |      100 |     100 |     100 | 
  constants.ts                |   83.33 |      100 |     100 |     100 | 
  url.ts                      |    92.3 |      100 |     100 |     100 | 
 src/hooks                    |   96.42 |    66.66 |    90.9 |     100 | 
  useAuthenticated.ts         |     100 |      100 |     100 |     100 | 
  useToast.ts                 |   95.23 |        0 |      90 |     100 | 10
 src/pages/[redirect]         |   86.11 |       75 |   66.66 |   90.32 | 
  index.tsx                   |   86.11 |       75 |   66.66 |   90.32 | 31-32,38
 src/pages/app                |     100 |      100 |     100 |     100 | 
  index.tsx                   |     100 |      100 |     100 |     100 | 
 src/pages/dashboard          |     100 |      100 |     100 |     100 | 
  index.tsx                   |     100 |      100 |     100 |     100 |                   
 src/services                 |     100 |      100 |     100 |     100 | 
  api.tsx                     |     100 |      100 |     100 |     100 | 
 src/utils                    |     100 |       90 |     100 |     100 | 
  formatDate.ts               |     100 |    88.88 |     100 |     100 | 7
  validateUrl.ts              |     100 |      100 |     100 |     100 | 
------------------------------|---------|----------|---------|---------|-------------------
## Additional Notes

No additional considerations.

### CODE REVIEW

**Comment by skv93-coder** [comment]
Can you add screenshots of ui

**Comment by skv93-coder** [comment]
Lgtm

**Review by skv93-coder — DISMISSED** [review]

File: src/components/Navbar/index.tsx [hunk 1]
```diff
@@ -39,8 +40,9 @@ const Navbar = () => {
         <>
             <nav className="bg-gray-900 p-4 h-[8vh]">
                 <div className="flex items-center justify-between">
-                    <Link href="/" className="text-white text-2xl font-bold">
-                        URL Shortener
+                    <Link href="/" className="flex">
+                        <Image src="/rds.png" alt="logo" width={30} height={30} className="mr-2 w-auto" />
```
Comment (ajoykumardas12) [inline src/components/Navbar/index.tsx:44]: The file name can be different I think, since it's not RDS logo? 'rdsli.png' maybe?

Comment (sahsisunny) [reply src/components/Navbar/index.tsx]: Done

**Review by ajoykumardas12 — APPROVED** [review]

**Pull request merged**